kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_with_constraints.h

```diff
@@ -219,7 +219,7 @@ class ResidualBasedEliminationBuilderAndSolverWithConstraints
 
     void SetUpSystem(ModelPart& rModelPart) override
     {
-        if(rModelPart.MasterSlaveConstraints().size() > 0)
+        if(ConstraintUtilities::NumberOfActiveConstraints(rModelPart) > 0)
```

Comment:
how much is the performance penalty from looping?
I.e. if you have lots of constraints I can imagine this to become expensive too since is done many many times